src/app/filters/filter-field.ts

```diff
@@ -12,6 +12,7 @@ import { FilterQuery } from './filter-query';
 export class FilterField {
   id?: string;
   placeholder?: string;
+  primaryQueries?: FilterQuery[];
```

Comment:
Looking at the type-ahead example, the primaryQueries property isn't used. Without an example, I can't determine exactly how it is intended to work -- we need an example. 

That said, I don't believe we should have both primaryQueries and queries in FilterField. It's confusing how this should be used, how options are ordered, etc. I had a difficult time understanding what you were doing here. 

Please simplify this by using one list and add a property indicating that the option is not "filterable". This would be similar to marking an option as disabled or as a separator -- we don't have separate lists for that.